content/chainguard/libraries/java/build-configuration.md

```diff
@@ -416,14 +416,132 @@ allprojects {
+Libraries for Java. If the administrator for your organization’s repository
+manager created a new repository or virtual repository, you must update your
+Bazel configuration to use the repository manager.
+
+Bazel uses `MODULE.bazel` files to define external dependencies as `artifacts`.
+You can configure a Maven repository for artifact retrieval using `repositories`
+from the
+[rules_jvm_external](https://github.com/bazel-contrib/rules_jvm_external) rule: 
+
+Following is an example configuration for a repository manager:
+
+```MODULE.bazel
+bazel_dep(name = "rules_jvm_external", version = "6.3")
+maven = use_extension("@rules_jvm_external//:extensions.bzl", "maven")
+
+maven.install(
+    name = "maven",
+    # Example dependencies to retrieve
+    artifacts = [
+        "com.google.guava:guava:32.0.1-jre",
+        "org.slf4j:slf4j-api:2.0.5", 
+        "ch.qos.logback:logback-classic:1.4.7",
+    ],
+    repositories = [
+        # To use Chainguard Libraries for Java via a repository manager:
+        "https://repo.example.com/repository/chainguard-maven/",
+    ],
+    # Uncomment and configure authentication if needed:
+    # auth = {
+    #     "https://repo.example.com/repository/chainguard-maven/": {
+    #         "type": "basic",
+    #         "username": "YOUR_USERNAME_FOR_REPOSITORY_MANAGER", 
+    #         "password": "YOUR_PASSWORD",
+    #     },
+    # },
+)
+
+use_repo(maven, "maven")
+```
+
+Example URLs for repository managers:
+
+* Cloudsmith: https://dl.cloudsmith.io/basic/exampleorg/chainguard-maven/maven/
+* JFrog Artifactory: https://example.jfrog.io/artifactory/chainguard-maven/
+* Sonatype Nexus: https://repo.example.com:8443/repository/chainguard-maven/
+
+If your organization does not use a repository manager, you can configure the
+Chainguard Libraries for Java repository directly, and include the Maven Central
+repository as fallback. Replace the placeholders `CHAINGUARD_JAVA_IDENTITY_ID`
+and `CHAINGUARD_JAVA_TOKEN` with the credentials provided by Chainguard:
+
+```MODULE.bazel
+maven.install(
+    name = "maven",
+    # Example dependencies to retrieve
+    artifacts = [
+        "com.google.guava:guava:32.0.1-jre",
+        "org.slf4j:slf4j-api:2.0.5", 
+        "ch.qos.logback:logback-classic:1.4.7",
+    ],
+    
+    repositories = [
+        # To use Chainguard Libraries directly (requires credentials):
+        "https://libraries.cgr.dev/java/",
+        
+        # Use Maven Central as fallback
+        "https://repo1.maven.org/maven2/",
+    ],
+    auth = {
+         "https://libraries.cgr.dev/java/": {
+             "type": "basic",
+             "username": "CHAINGUARD_JAVA_IDENTITY_ID", 
+             "password": "CHAINGUARD_JAVA_TOKEN",
+         },
+    },
+)
+```
+
+Ensure that the Chainguard repository is listed before any other repositories to
+prioritize it for artifact retrieval. 
+
+For more complex Bazel setups, you can use a `.netrc` file containing the
+credentials:
+
+```
```

Comment:
I had ".netrc" here but on rendering it changed to ".Netrc" which is wrong .. so I removed..